demos/dermatology.html

```diff
@@ -296,7 +296,9 @@
                     currWidget = widget;
                     if (params) {
                         for (var key in params) {
-                            currWidget[key](params[key]);
+                            if(typeof(currWidget[key]) === "function"){

```

Comment:
i suspect there are other areas of of code that can use this same fix in the library?